app/db_accessor.py

```diff
@@ -334,14 +334,13 @@ def editMember(updated_member, old_member):
     if demographic_data['income'] != old_demographic_data['income']:
         user.member[0].income = demographic_data['income']
     if demographic_data['credit_score'] == '':
-        demographic_data['credit_score'] = None
-    if demographic_data['credit_score'] != old_demographic_data['credit_score']:
+        user.member[0].credit_score = 0
+    elif demographic_data['credit_score'] != old_demographic_data['credit_score']:
```

Comment:
just use the line: `if 'credit_score' in demographic_data` to check if the key exists
So you wouldn't need an if and elif.

Just modify what you have in your 'elif' right now to 
`if 'credit_score' in demographic_data and demographic_data['credit_score'] != old_demographic_data['credit_score']`

Same thing for has_car and anything else which might cause this issue

Reply:
Sometimes demographic_data['credit_score'] is returning '' when it is empty causing int() to error. So credit_score can exist in demo_data, but be empty. I think I'll have to keep both lines, but check that credit_score is in demo_data for each.